src/beam.ts

```diff
@@ -635,25 +684,28 @@ export class Beam extends Element {
 
     for (let i = 0; i < notes.length; ++i) {
       const note = notes[i];
-      const stemX = note.getStemX();
-      const { topY: stemTipY } = note.getStemExtents();
-      const beamedStemTipY = this.getSlopeY(stemX, firstStemX, firstStemTipY, slope) + y_shift;
-      const preBeamExtension = note.getStem().getExtension();
-      const beamExtension = stem_direction === Stem.UP ? stemTipY - beamedStemTipY : beamedStemTipY - stemTipY;
-
-      note.stem.setExtension(preBeamExtension + beamExtension);
-      note.stem.renderHeightAdjustment = -Stem.WIDTH / 2;
-
-      if (note.isRest() && show_stemlets) {
-        const beamWidth = beam_width;
-        const totalBeamWidth = (beam_count - 1) * beamWidth * 1.5 + beamWidth;
-        note.stem.setVisibility(true).setStemlet(true, totalBeamWidth + stemlet_extension);
+      const stem = note.getStem();
+      if (stem) {
```

Comment:
Add an `else` here to throw an exception, so it doesn't silently fail.

Reply:
ok